.gitignore

```
@@ -536,7 +536,7 @@ MigrationBackup/
 # Backup folder for Package Reference Convert tool in Visual Studio 2017
 
 # End of https://www.gitignore.io/api/csharp,powershell,dotnetcore,visualstudio,visualstudiocode
-deploy/**
+Win64_Shipping_Client/**
```

This shouldn't exist in the repo as mb2 is a link
```suggestion
deploy/**
```

---

fixed